core/lib/build.rs

```diff
@@ -3,19 +3,19 @@
 use yansi::{Paint, Color::{Red, Yellow}};
 
 fn main() {
-    const MIN_VERSION: &'static str = "1.46.0";
+    const MIN_VERSION: &str = "1.46.0";
 
     if let Some(version) = version_check::Version::read() {
         if !version.at_least(MIN_VERSION) {
             let msg = "Rocket requires a more recent version of rustc.";
             eprintln!("{} {}", Red.paint("Error:").bold(), Paint::new(msg).bold());
             eprintln!("Installed version: {}", Yellow.paint(format!("{}", version)));
-            eprintln!("Minimum required:  {}", Yellow.paint(format!("{}", MIN_VERSION)));
+            eprintln!("Minimum required:  {}", Yellow.paint(MIN_VERSION.to_string()));
```

Comment:
I don't think we need `to_string()` at all, actually. Same with the `format!` above.

Reply:
Good catch!